crates/agent/src/discovers.rs

```diff
@@ -124,31 +131,42 @@ impl DiscoverHandler {
             .await
             .context("clearing old errors")?;
 
+        let mut cmd = async_process::Command::new(format!("{}/flowctl-go", &self.bindir));
+        cmd.arg("api")
+            .arg("discover")
+            .arg("--config=/dev/stdin")
+            .arg("--image")
+            .arg(&image_composed)
+            .arg("--network")
+            .arg(&self.connector_network)
+            .arg("--output=json")
+            .arg("--log.level=warn")
+            .arg("--log.format=color");
+        if self.allow_local {
+            cmd.arg("--allow-local");
```

Comment:
`flowctl-go` supports running local connectors? I thought it didn't, and that only the Rust runtime did.. 🤔